solr/core/src/test/org/apache/solr/schema/DenseVectorFieldTest.java

```diff
@@ -456,6 +458,25 @@ public void indexing_correctDocument_shouldBeIndexed() throws Exception {
     }
   }
 
+  @Test
+  public void indexing_highDimensionalityVectorDocument_shouldBeIndexed() throws Exception {
+    try {
+      initCore("solrconfig-basic.xml", "schema-densevector-high-dimensionality.xml");
+
+      List<Float> highDimensionalityVector = new ArrayList<>();
+      for(float i=0; i< 2048f; i++){
+        highDimensionalityVector.add(i);
+      }
+      SolrInputDocument correctDoc = new SolrInputDocument();
+      correctDoc.addField("id", "0");
+      correctDoc.addField("vector", highDimensionalityVector);
+
+      assertU(adoc(correctDoc));
```

Comment:
And query for it?

Reply:
added a first rough implementation in the latest commit, it's just a first step, happy to refine the test later